manifests/r/RicoSuter/NSwagStudio/13.14.7/RicoSuter.NSwagStudio.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RicoSuter.NSwagStudio
+PackageVersion: 13.14.7
```

```suggestion
PackageVersion: 13.14.7.0
```
update your automation to add `.0` to the version.